code/modules/clothing/masks/gasmask.dm

```diff
@@ -154,6 +154,142 @@
+			recent_uses = initial(recent_uses)
+
+		switch(recent_uses)
+			if(3)
+				to_chat(usr, "<span class='warning'>\The [src] is starting to smell weird.</span>")
+			if(4)
+				to_chat(usr, "<span class='userdanger'>\The [src] is smelling like burnt peanuts!</span>")
+			if(5) //overload
+				broken_hailer = TRUE
+				to_chat(usr, "<span class='userdanger'>\The [src]'s the depleted banana module shorts.</span>")
+				new /obj/item/grown/bananapeel(src)
+				return
+
+		phrase = rand(1,18)
+
+		if(gorilla)
+			phrase_text = "FUCK YOUR CUNT YOU SHIT EATING COCKSTORM AND EAT A DONG FUCKING ASS RAMMING SHIT FUCK EAT PENISES IN YOUR FUCK FACE AND SHIT OUT ABORTIONS OF FUCK AND POO AND SHIT IN YOUR ASS YOU COCK FUCK SHIT MONKEY FUCK ASS WANKER FROM THE DEPTHS OF SHIT."
+			phrase_sound = "emag"
+		else
+
+			switch(phrase)	//sets the properties of the chosen phrase
+				if(1)				// good cop
+					phrase_text = "HALT! HALT! HALT!"
+					phrase_sound = "monkey2"
+				if(2)
+					phrase_text = "Stop in the name of the Law."
+					phrase_sound = "monkey1"
+				if(3)
+					phrase_text = "Compliance is in your best interest."
+					phrase_sound = "monkey5"
+				if(4)
+					phrase_text = "Prepare for justice!"
+					phrase_sound = "monkey3"
+				if(5)
+					phrase_text = "Running will only increase your sentence."
+					phrase_sound = "monkey4"
+				if(6)				// bad cop
+					phrase_text = "Don't move, Creep!"
+					phrase_sound = "monkey1"
+				if(7)
+					phrase_text = "Down on the floor, Creep!"
+					phrase_sound = "monkey3"
+				if(8)
+					phrase_text = "Dead or alive you're coming with me."
+					phrase_sound = "monkey2"
+				if(9)
+					phrase_text = "God made today for the crooks we could not catch yesterday."
+					phrase_sound = "monkey1"
+				if(10)
+					phrase_text = "Freeze, Scum Bag!"
+					phrase_sound = "monkey4"
+				if(11)
+					phrase_text = "Stop right there, criminal scum!"
+					phrase_sound = "monkey5"
+				if(12)				// LA-PD
+					phrase_text = "Stop or I'll bash you."
+					phrase_sound = "monkey3"
+				if(13)
+					phrase_text = "Go ahead, make my day."
+					phrase_sound = "monkey2"
+				if(14)
+					phrase_text = "Stop breaking the law, ass hole."
+					phrase_sound = "monkey4"
+				if(15)
+					phrase_text = "You have the right to shut the fuck up."
+					phrase_sound = "monkey2"
+				if(16)
+					phrase_text = "Shut up crime!"
+					phrase_sound = "monkey5"
+				if(17)
+					phrase_text = "Face the wrath of the golden bolt."
+					phrase_sound = "monkey1"
+				if(18)
+					phrase_text = "I am, the LAW!"
+					phrase_sound = "monkey3"
+
+		usr.audible_message("[usr]'s Banana-tor:")
+		var/list/hearers = get_hearers_in_view(DEFAULT_MESSAGE_RANGE, src)
+		var/datum/language/monkeyspeak = GLOB.language_datum_instances[/datum/language/monkey]
+		for(var/mob/hear in hearers)
+			if(hear.has_language(/datum/language/monkey))
+				if(gorilla)
+					to_chat(hear, "<font color='brown' size='6'><b>[phrase_text]</b></font>")

```

Comment:
This will get sent to deaf people, do this instead
```suggestion
					hear.show_message("<font color='brown' size='6'><b>[phrase_text]</b></font>", MSG_AUDIBLE)
```